core/src/test/java/com/novoda/noplayer/internal/exoplayer/forwarder/ExoPlayerErrorMapperTest.java

```diff
@@ -0,0 +1,122 @@
+package com.novoda.noplayer.internal.exoplayer.forwarder;
+
+import android.media.MediaCodec;
+import android.net.Uri;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ParserException;
+import com.google.android.exoplayer2.drm.DrmSession;
+import com.google.android.exoplayer2.upstream.DataSpec;
+import com.novoda.noplayer.NoPlayer;
+import com.novoda.noplayer.PlayerErrorType;
+import com.novoda.noplayer.drm.StreamingModularDrm;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+
+import static com.google.android.exoplayer2.upstream.HttpDataSource.InvalidResponseCodeException;
+import static org.fest.assertions.api.Assertions.assertThat;
+
+@RunWith(Enclosed.class)
+public class ExoPlayerErrorMapperTest {
+
+    public static class GivenSourceException {
+        @Test
+        public void InvalidResponseCodeException_mapsTo_InvalidResponseCode() {
```

Comment:
Uppercase 😬

Reply:
It is upper case because it is the name of the class `InvalidResponseCode`